docs/docker_container.md

```diff
@@ -303,7 +303,28 @@ docker run --rm -d -p 9001:9001 \
 -e S3_ENDPOINT=“${S3_ENDPOINT}” \
 openvino/model_server:latest \
 --model_path s3://bucket/model_path --model_name s3_model --port 9001
+```
```

Comment:
it might be useful to add that s3_endpoint is optional for AWS s3 storage and mandatory for Minio and other s3 compatible storage types